frontend/package.json

```diff
@@ -90,7 +90,6 @@
     "simplebar": "^5.3.5"
   },
   "devDependencies": {
-    "@axe-core/react": "^4.2.2",
```

Comment:
if we remove axe here;
Can you remove the paragraph mentioning it in the frontend README.md?
We can replace it with a few sentences about lighthouse accessibility checks.